pkg/repo/index.go

```diff
@@ -76,32 +74,139 @@ func (c ChartVersions) Less(a, b int) bool {
 	return i.LessThan(j)
 }
 
-// IndexFile represents the index file in a chart repository
-type IndexFile struct {
+// ChartRepositoryIndex represents the index file in a chart repository
+type ChartRepositoryIndex struct {
```

Comment:
It's not necessary, I just tried to change it to be more explicit. Reverted in https://github.com/kubernetes/helm/pull/1766/commits/f8a261f277d29d5ccff177b5fe6cf3781301fd70